mne/export/_export.py

```diff
@@ -15,8 +15,10 @@ def export_raw(fname, raw, fmt='auto', physical_range='auto',
     """Export Raw to external formats.
 
     Supported formats:
-    - EEGLAB (set, uses :mod:`eeglabio`)
-    - EDF (edf, uses ``EDFlib-Python``)
+
+        - EEGLAB (set, uses :mod:`eeglabio`)
+        - EDF (edf, uses ``EDFlib-Python``)
```

Comment:
```suggestion
        - EDF (.edf, uses ``EDFlib-Python``)
```

No `:mod:` prefix here?

Reply:
EDFlib-Python doesn't have a sphinx generated documentation, so I don't think it would work?